pynetbox/models/dcim.py

```diff
@@ -151,6 +151,7 @@ def __str__(self):
 
 
 class Interfaces(TraceableRecord):
+    device = Devices
```

Comment:
Can you put this in a separate PR, it doesn't look like a speed improvement but a bug fix (if a bug fix and there isn't an issue for it, please open an issue).